### ISSUE TITLE
A11y Code Sprint Berlin: Success Criterion 9.1.3.1a - Info and Relationships - Headings in content elements

### ISSUE THREAD

**Issue by alexakressmann** [benjaminkott/bootstrap_package#1318]
**Page No 1**
-
https://bitv-v12.live.typo3.cps.321.works/content-examples/interactive/accordion

**Description**
-
All headings need to be marked as headings in correct order.
[To WCAG](https://www.w3.org/TR/WCAG21/#info-and-relationships)

**Solution**
-
1. Insert visible H2: "Media representation in the accordion"
2. Convert accordion entries from H4 into H3

**Ideal Solution**
-
Heading levels in the accordion should be editorially adaptable in the backend of TYPO3 by editors.

---

**Page No 2**
-
https://bitv-v12.live.typo3.cps.321.works/content-examples/media/icon-group

**Description**
-
All headings need to be marked as headings in correct order.
[To WCAG](https://www.w3.org/TR/WCAG21/#info-and-relationships)

**Solution**
-
1. Add H2: "Display options in the icon group"
2. Convert headings from H4 into H3

**Ideal solution**
-
Heading levels in the icon group should be editorially adaptable in the backend of TYPO3 by editors.

---

**Page No 3**
-
https://bitv-v12.live.typo3.cps.321.works/content-examples/interactive/timeline

**Description**
-
All headings need to be marked as headings in correct order.
[To WCAG](https://www.w3.org/TR/WCAG21/#info-and-relationships)

**Solution**
-
Convert headings from H4 into H3

**Ideal solution**
-
Heading levels in the timeline should be editorially adaptable in the backend of TYPO3 by editors.

**Comment by alboth** [comment]
ich @alboth schnapp mit das ticket. @alexakressmann kannst du es bitte auf "in progress" stellen. danke!

**Comment by alboth** [comment]
@alexakressmann ich habe hier noch redebedarf. ich denke wir fixen hier nur den EINEN case auf der Demo Seite. 
Sobald Redakteur:in das Accordion mal tiefer einsetzt auf in einem Dokument, wäre die H3 falsch und die H4 plötzlich wieder korrekt.... die H4 ist aber zZ fest vercoded und nicht einstellbar.

Lösungsvorschläge @benjaminkott @alexakressmann 

1. Eine **Einstellung/Feld schaffen im BE**, damit das am ContentElement eingestellt werden kann von Redkateur:in
2. Das **Header-Level des Accordion immer +1 zum `header_layout`** des umschließenden Content-Elements. Allerdings gibt es keine klare aussage in den Fällen wo A: Kein Header gesetzt wurde, der Header auf Versteckt gesetzt wurde und ggf sogar wenn der Header auf Default steht

**Comment by alboth** [comment]
@alexakressmann bitte nimm mich von dem ticket wieder runter und schau, dass die redakteure jetzt ALLE o.g. elemente nachpflegen mit der neuen technik die im BE zur verfügung steht. 
da bitte kritisch über die GANZE seite nochmal schauen, ob ggf diese elemente irgendwo verwendet werden

**Comment by alexakressmann** [comment]
Who can take this task?

**Comment by alexakressmann** [comment]
Ralf, a few changes are made yet, but for the rest you have to look in the backend if there are new possibilites to set the headings. Please have a look in the backend if you can change the headings as we mentioned in the issue in the parts "solution". For a quick check you can use the browser add-on headings map.

**Comment by RalleHo** [comment]
All 3 pages are modified.

**Comment by alexakressmann** [comment]
QA successful
